nf_core/lint_utils.py

```diff
@@ -38,7 +38,9 @@ def print_fixes(lint_obj, module_lint_obj):
     """Prints available and applied fixes"""
 
     if len(lint_obj.could_fix):
-        fix_cmd = "nf-core lint {} --fix {}".format(lint_obj.wf_path, " --fix ".join(lint_obj.could_fix))
+        fix_cmd = "nf-core lint {}--fix {}".format(
+            "" if lint_obj.wf_path == "." else f"--dir {lint_obj.wf_path}", " --fix ".join(lint_obj.could_fix)
```

Comment:
Could in theory just take the command used and append fixed I think. But not sure it's worth the effort, this looks good too.